payjoin/src/uri.rs

```diff
@@ -2,27 +2,49 @@ use std::borrow::Cow;
 use std::convert::TryFrom;
 
 use bitcoin::address::{Error, NetworkChecked, NetworkUnchecked};
-use bitcoin::Network;
+use bitcoin::{Address, Amount, Network};
 use url::Url;
 
+/// Payjoin enum represents a payjoin object.
```

Comment:
Something like "payjoin parameters for bip21" would be much more descriptive than the ambiguous "object"